bundles/azure/aso/psql/bundle/config/00-schema.yml

```diff
@@ -0,0 +1,143 @@
+#@data/values-schema
+---
+#@schema/title "ASO Controller Namespace"
+#@schema/desc "The Namespace where the Azure ASO controller is installed that should own this Azure RM resource"
+aso_controller_namespace: ""
+
+#@schema/title "ResourceName"
+#@schema/desc "Name for the resources"
```

Comment:
_nit_: that's a bit of a lie. Some resources are named like that, others aren't. I think we might want to change that to some thing like `Name of the Postgres Server Instance", because I'd assume that what a user (a service operator) would care most about.

Reply:
See: https://github.com/vmware-tanzu/tanzu-application-platform-reference-service-packages/pull/22/files#r975346168